tests/erp/mapper/test_vaccination.py

```diff
@@ -0,0 +1,164 @@
+import json
+import pytest
+
+from datetime import datetime
+
+from django.contrib.gis.geos import Point
+
+from erp.mapper.vaccination import RecordMapper
+from erp.models import Activite, Erp
+
+from tests.erp.mapper.fixtures import neufchateau, sample_record_ok
+
+
+@pytest.fixture
+def activite_cdv(db):
+    return Activite.objects.create(nom="Centre de vaccination")
+
+
+def test_init():
+    with pytest.raises(RuntimeError) as err:
+        RecordMapper({})
+    assert "Propriété manquante 'geometry'" in str(err.value)
+
+    with pytest.raises(RuntimeError) as err:
+        RecordMapper({"geometry": []})
+    assert "Propriété manquante 'properties'" in str(err.value)
+
+
+def test_source_id():
+    m = RecordMapper({"geometry": [], "properties": {"c_gid": 123}})
+    assert m.source_id == "123"
+
+
+def test_source_id_missing():
+    with pytest.raises(RuntimeError) as err:
+        m = RecordMapper({"geometry": [], "properties": {"c_gid": None}})
+        m.source_id
+    assert "Champ c_gid manquant" in str(err.value)
+
+
+def test_skip_importing_closed(activite_cdv, neufchateau, sample_record_ok):
+    sample_closed = sample_record_ok.copy()
+    sample_closed["properties"]["c_date_fermeture"] = "2021-01-01"
+
+    m = RecordMapper(sample_closed, today=datetime(2021, 1, 2))
+    with pytest.raises(RuntimeError) as err:
+        m.process(activite_cdv)
+
+    assert "SKIPPED: Centre fermé le 2021-01-01" in str(err.value)
+
+
+def test_skip_importing_reserve_ps(activite_cdv, neufchateau, sample_record_ok):
```

Comment:
Same as above, using "pros" instead of "ps" would be more explicit imho

Reply:
Addressed in https://github.com/MTES-MCT/acceslibre/pull/136/commits/71bc108f68b7438bafd66b7e206b0167e3ee0452